azure-iot-device/tests/common/test_mqtt_transport.py

```diff
@@ -378,6 +379,18 @@ def test_client_raises_unexpected_error(
             transport.connect(fake_password)
         assert e_info.value.__cause__ is arbitrary_exception
 
+    @pytest.mark.it(
+        "Raises a ConnectionFailedException if Paho connect raises a socket.error Exception"
```

Comment:
>ConnectionFailedException [](start = 18, length = 25)

should this be a c`onnectionfailederror` or a `connectionfailederror` exception ?